### PR TITLE
made tweaks to the settingsmeta

### PR DESCRIPTION
**Pull request by Mn0491**
made tweaks to the settingsmeta so that it is more clear on web ui

### CODE REVIEW

**Comment by penrods** [comment]
I was gonna get around to several things on this skill, but since you are starting this we can wrap them up here:

* You tell them to use "Green", but I don't think that actually works.  The uppercase 'Green' isn't found in the dictionary.  :)

* As a general comment, if you refer to a value like the README.md you should go ahead and make it a clickable link.  Although the next point probably skips doing that...

* I'd like to go ahead and switch to using translate_namedvalues() (see https://github.com/MycroftAI/skill-wink-iot/blob/master/__init__.py#L95).   This will become a standard method in MycroftSkill next release, but you could implement it in the skill like I did there, then we'll remove it later.  Then I'd like the list to be the whole list of named web colors (e.g. "Chartreuse, Light Pink, etc.).  This mechanism should make it pretty easy to do:
```
     red, FF0000
     green, 00FF00
     blue, 0000FF
     chartreuse, 7FFF00
     ...
```
* What do you think about using fuzzy_match() against the color?  So if they say something like "orange red" or "orangish red" or "orangy red" it might work?  Just a thought.

* I've gotten stuck in the Brightness states and can't say cancel or anything.  Can you either add that or maybe switch to the new mechanism that you and Matthew just added for scripting dialog?  (Might be out of the scope of this PR.)

**Comment by Mn0491** [comment]
Fixed up a bunch of stuff!

- color name for web UI is not case sensitive anymore

- Clickable README.md

- translate_namevalues in place

- fuzzy matching

- fixed up some converse UX

- change settingsmeta to be more clear and more informative

@penrods let me know how this looks. And please Merge and Squash so we can get rid of all these generic commits

**Comment by Mn0491** [comment]
@penrods did you ever figure out the issue with the command "set eye color to blue" not working after the change?

**Comment by penrods** [comment]
I just debugged into this some and don't see any obvious reason why it should be failing.  I didn't attempt to dig into Padatious.  Anything obvious being done incorrectly here, @MatthewScholefield ?

The color.entity file contains lots of color names, but other than that nothing strange that I can see.  Using the phrase "change your eye color to red" should perfectly match the first first intent in the eye.color.intent file.  I didn't debug into the Padatious code, but I did try a shorter color.entity list; removing the training empty line; and switching to double-moustache syntax -- none changed anything, the match message.data still just contains an utterance and no 'color' entity match.

Ideas?

File: vocab/en-us/eye.color.intent [hunk 1]
```diff
@@ -1,4 +1,11 @@
+change your eye color to (the |){color|*}
 change the eye color to (the |){color}
-set eye color to (the |){color}
-have eye color be (the |){color}
-change eye to (the |){color}
\ No newline at end of file
+set eye color to (the |){color|*}
```
Comment (MatthewScholefield) [inline vocab/en-us/eye.color.intent]: This syntax isn't supported: `{color|*}`
You can either rewrite it as `{color}` or if you want to give room for unknown words as a color, add the following as a line in the `.entity` file: `:0`. It will train it to still activate 100% on unknown words.

File: vocab/en-us/eye.color.intent [hunk 2]
```diff
@@ -1,4 +1,11 @@
...
+change eye to (the |){color}
+set eye color to something {color}
+change eye color to a {} color
```
Comment (MatthewScholefield) [inline vocab/en-us/eye.color.intent]: `{}` should probably be `{color}`.

**Comment by penrods** [comment]
More info, @MatthewScholefield:

I diddled around using double-moustaches and got these phrases to work   :-)
set the eye color to purple
>> That's a good color. Look at me now.

change the eye color to purple
>> That's a good color. Look at me now.


This asks the follow up because there is no 'color' entity  :-/
change your eye color to purple
>> Name a color to try.

This just fails to match  :-(
set your eye color to purple
>> Can you rephrase that?

(All of this was with double-moustaches on the intents, e.g.:
```
change your eye color to (the |){{color|*}}
change the eye color to (the |){{color}}
set eye color to (the |){{color|*}}
change eye to (the |){{color}}
set eye color to something {{color}}
change eye color to a {{}} color
set eye to a {{color}} color
set eye to some {{color}} color
change eye to a {{color}} color
set eye color
change eye color
```

THEN I reverted to single moustaches and behavior change totally.  The only working request was:
```change the eye color to purple```.  All other versions resulted in "I don't know what that means".

Are single vs double {color} / {{color}} different intentionally?  Padatious quirk or bug?

File: vocab/en-us/eye.color.intent [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+set eye to a {color} color
+set eye to some {color} color
+change eye to a {color} color
+set eye color
+change eye color
```
Comment (MatthewScholefield) [inline vocab/en-us/eye.color.intent]: As a side note, I noticed from my parents that it's common to say things like `(change | set) your eyes to {color}`.

**Comment by MatthewScholefield** [comment]
@penrods Padatious has always used single brackets. We should probably support to double brackets to be consistent. Right now it interprets that as an entity whose name is `{color}` rather than the current entity name of `color`.
